src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -3192,7 +3217,8 @@ void SvgLoader::run(unsigned tid)
 
         _updateStyle(loaderData.doc, nullptr);
     }
-    root = svgSceneBuild(loaderData.doc, vx, vy, vw, vh, w, h, align, meetOrSlice, svgPath);
+    root = svgSceneBuild(loaderData.doc, vx, vy, vw, vh, w, h, align, meetOrSlice, svgPath, viewFlag);
+    applySvgViewFlag(root.get(), vx, vy, vw, vh, w, h, viewFlag);
```

Comment:
How about put this call in svgSceneBuild inside()? otherwise we don't need to expose the method.

Reply:
```if (!isBBoxGiven) this->done();``` - removed, also after changes it wasn;t necessary any more

applySvgViewFlag -> internal fun of the scene biulder, but now svgSceneBuild changes the vx, vy, vw, vh values